swssconfig/swssconfig.cpp

```diff
@@ -0,0 +1,203 @@
+#include <stdlib.h>
+#include <iostream>
+#include <vector>
+#include "logger.h"
+#include <fstream>
+#include "dbconnector.h"
+#include "producertable.h"
+#include "json.hpp"
+
+using namespace std;
+using namespace swss;
+using json = nlohmann::json;
+
+int db_port							= 6379;
+const char* const hostname 			= "localhost";
```

Comment:
This behavior is based on neighsyncd as a sample. Most of apps use hardcoded port and host name.
If we decide to change all of them it can be done in one changelist.